pkg/sql/vars.go

```diff
@@ -831,8 +831,11 @@ var varGen = map[string]sessionVar{
 	// See https://www.postgresql.org/docs/10/static/runtime-config-client.html#GUC-LOC-TIMEOUT
 	`lock_timeout`: makeCompatIntVar(`lock_timeout`, 0),
 
+	// See https://www.postgresql.org/docs/9.1/runtime-config-compatible.html
```

Comment:
nit: use the latest version of the postgresql docs.
https://www.postgresql.org/docs/13/runtime-config-compatible.html